server/controller/WebHookListeners.js

```diff
@@ -1,12 +1,13 @@
 import RelayController from './RelayController';
 import SlackTriggerDispatcher from '../thirdparty/controller/SlackTriggerDispatcher';
+import { performGithubActionOnSlackTrigger } from '../thirdparty/controller/githubHandler';
 
 const slackWebHookListener = async (event, authedUsers) => {
-  const slackTriggeredRelays = await RelayController.getRunningRelaysWithTriggerApp('Slack');
-
-  if (slackTriggeredRelays.length !== 0) {
-    setImmediate(SlackTriggerDispatcher, slackTriggeredRelays, event, authedUsers);
-  }
+  //const slackTriggeredRelays = await RelayController.getRunningRelaysWithTriggerApp('Slack');
+  if (event && event.text && event.text.split(':')[0] === 'github') performGithubActionOnSlackTrigger(event, authedUsers);
```

Comment:
This breaks slack to slack relays

Reply:
Uncommenting the line will fix the slack to slack relays.